configure

```diff
@@ -590,8 +590,8 @@ if (!$ace_src || !$tao_src) {
 
     $ace_src = normalizePath('ACE_TAO/ACE');
     $tao_src = normalizePath('ACE_TAO/TAO');
-
-  } else {
+  }
+  else {
```

Comment:
I wouldn't have guessed this was a recommend style, is this a Perl convention?

Reply:
Yes, it matches MPC and probably other Perl projects.